pkgs/by-name/hy/hyprlock/package.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, libGL
+, libxkbcommon
+, hyprlang
+, pam
+, wayland
+, wayland-protocols
+, cairo
+, pango
+, unstableGitUpdater
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hyprlock";
+  version = "0-unstable-2024-02-20";
```

Comment:
Is `0-` some new convention?

Reply:
<https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming>

Reply:
So this is the successor of `0.pre+date=YYYY-MM-DD`. Got it.